packages/clay-color-picker/src/Hue.tsx

```diff
@@ -20,66 +20,56 @@ type Props = {
 	value: number;
 };
 
-const useIsomorphicLayoutEffect =
-	typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect;
-
 /**
  * Renders Hue component
  */
 const ClayColorPickerHue = ({value = 0, onChange = () => {}}: Props) => {
 	const containerRef = React.useRef<HTMLDivElement>(null);
-	const selectorActive = React.useRef<boolean>(false);
 
-	const {onPointerMove, setXY, x, y} = usePointerPosition(containerRef);
+	const {setXY, y} = usePointerPosition(containerRef);
 
-	const removeListeners = () => {
-		selectorActive.current = false;
+	const [internalValue, setInternalValue] = useState<number>(value);
 
-		window.removeEventListener('pointermove', onPointerMove);
-		window.removeEventListener('pointerup', removeListeners);
-	};
+	const handleOnChangeEnd = (event: any) => {
```

Comment:
Since we won't have the internal state anymore, just add the `onChange` callback as a property of the `ClaySlider`, we won't need that anymore because we will be directly updating the parent state instead of the internal one.